front-end/src/interfaces/audit.ts

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable semi */
 
+import type { VulnerabilitySeverity } from '@/sdk/src/types';
+
 export default interface IAudit {
   title: string;
-  severity: string;
+  severity: VulnerabilitySeverity;
   description: string;
 }
```

Comment:
You should also have Vulnerability type from sdk that looks exactly like IAudit

Reply:
Gotcha!